### DISCUSSION TITLE
ImGui Viewport, bring imgui viewports to top front on main window click

### DISCUSSION THREAD

**Discussion by Im-Rises** [ocornut/imgui#7402] [New Users Build/Link/Run issues ONLY!]
Hello everyone
Using ImGui Viewports, is it possible to bring all the imgui windows front when clicking on the main glfw/sdl window?
Can't find a viewport or anything, this cause an issue when io.ConfigViewportsNoTaskBarIcon is set to true. The ImGui windows are hiding behind my other running programs and clicking on the glfw/sdl window doesn't bring them at the front before my other apps.
Thanks in advance!

**Comment by PathogenDavid** [comment]
For future reference, [this section is for build/link/run/font issues only](https://github.com/ocornut/imgui/discussions/4497). Next time [open an issue](https://github.com/ocornut/imgui/issues/new/choose).

Having this work as expected currently requires the platform backend to honor `ParentViewportID`.

Right now only `imgui_impl_win32` implements support for this feature.
